subgraph/packages/subgraph-allocations/schema.graphql

```diff
@@ -0,0 +1,20 @@
+type Schedule @entity {
+  id: ID!
+  durations: [BigInt!]!
```

Comment:
I'd suggest to store the UNIX timestamps for unlocks instead of durations for `Schedule`. Each unlock timestamp is calculated as `durations[i] + startTime`, where `startTime` can be read from the contract (`1630065600`)